mockOSApi.v2/Controllers/LoginController.cs

```diff
@@ -0,0 +1,71 @@
+using mockOSApi.Services;
+using Microsoft.AspNetCore.Mvc;
+using mockOSApi.DTO;
+
+/// <summary>
+/// Controller responsible with user authentication
+/// </summary>
+[ApiController]
+[Route("/api/[controller]")]
+[Produces("application/json")]
+public class AdminController : Controller
```

Comment:
The file is called LoginController, but the class is AdminController
There's a setting that lets you rename the class and the file at the same time

Reply:
Done